src/main/java/com/rapid7/ias/bamboo/impl/InsightAppSecScanTaskConfigurator.java

```diff
@@ -92,39 +105,54 @@ public Map<String,String> generateTaskConfigMap(@NotNull ActionParametersMap par
      */
     @Override
     public void validate(@NotNull ActionParametersMap params,
-                         @NotNull ErrorCollection errorCollection){
+                         @NotNull ErrorCollection errorCollection) {
         Logger log = Logger.getLogger(InsightAppSecScanTaskConfigurator.class);
         UtilityLogger logger = new UtilityLogger(log);
 
         super.validate(params, errorCollection);
         String regionValue = params.getString(SELECTED_REGION);
-        if (regionValue.equals("OTHER")){
+        if (regionValue.equals("OTHER")) {
             regionValue = params.getString(OTHER_REGION);
         }
 
         String credentialId = params.getString(SELECTED_CREDENTIAL);
         String appNameValue = params.getString(APP_NAME);
         String scanConfigNameValue = params.getString(SCAN_CONFIG_NAME);
+        String proxyHostValue = params.getString(PROXY_HOST);
+        String proxyPortValue = params.getString(PROXY_PORT);
+        String debugging = params.getString(DEBUGGING);
+        Boolean proxyEnabled = null;
 
         if (credentialId == null) {
             errorCollection.addError(SELECTED_CREDENTIAL, i18nBean.getText("error.apiKey"));
-        }else if (credentialsAccessor.getCredentials(Long.parseLong(credentialId)) == null) {
+        } else if (credentialsAccessor.getCredentials(Long.parseLong(credentialId)) == null) {
             errorCollection.addError(SELECTED_CREDENTIAL, i18nBean.getText("error.apiKeyNotExist"));
-        }else if (StringUtils.isEmpty(regionValue)){
+        } else if (StringUtils.isEmpty(regionValue)) {
             errorCollection.addError(SELECTED_REGION, i18nBean.getText("error.region"));
-        }else if(StringUtils.isEmpty(appNameValue)){
+        } else if (StringUtils.isEmpty(appNameValue)) {
             errorCollection.addError(APP_NAME, i18nBean.getText("error.appName"));
-        }else if(StringUtils.isEmpty(scanConfigNameValue)){
+        } else if (StringUtils.isEmpty(scanConfigNameValue)) {
             errorCollection.addError(SCAN_CONFIG_NAME, i18nBean.getText("error.scanConfigName"));
+        } else if (StringUtils.isNotEmpty(proxyPortValue) && !StringUtils.isNumeric(proxyPortValue)) {
+            errorCollection.addError(PROXY_PORT, i18nBean.getText("error.proxyPort"));
         }else{
             CredentialsData cred = credentialsAccessor.getCredentials(Long.parseLong(credentialId));
 
-            InsightAppSecHelper iasHelper = new InsightAppSecHelper(regionValue, cred.getConfiguration().get(CFG_PASSWORD), logger);
+            if (proxyHostValue.isEmpty()) {
+                proxyHostValue = proxyPortValue = null;
```

Comment:
Can we do this on multiple lines

proxyHostValue = null;
proxyPortValue = null;

This is valid, but not something ive seen done very often